entries/v/veem.com.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Veem": {
+    "domain": "veem.com",
+    "url": "https://www.veem.com/",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```